grails-app/views/job/show.gsp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+        <link rel="stylesheet" href="${resource(dir: 'css', file: 'imos.css')}" type="text/css">
+    </head>
+    <body>
+        <div class="imosHeader">
+            <div class="container">
+                <a class="btn" role="button" href="https://imos.aodn.org.au/imos123/home?uuid=8af21108-c535-43bf-8dab-c1f45a26088c">
+                    <img src="http://static.emii.org.au/images/logo/IMOS-Ocean-Portal-logo.png" alt="IMOS logo">
+                </a>
+            </div>
+        </div>
+        <div class="container">
+            <dl>
+                <g:labelledContent labelCode="job.id.label">${job.uuid}</g:labelledContent>
+                <g:labelledContent labelCode="job.createdTimestamp.label"><joda:format value="${job.createdTimestamp}" /></g:labelledContent>
+                <g:labelledContent labelCode="job.status.label">${job.status}</g:labelledContent>
+                <g:labelledContent labelCode="job.queuePosition.label">${job.queuePosition}</g:labelledContent>
+                <g:labelledContent labelCode="job.aggrUrl.label">${job.aggrUrl}</g:labelledContent>
+                <g:labelledContent labelCode="job.report.label"><pre>${job.report}</pre></g:labelledContent>
```

Comment:
I think the `<pre>` tag here is causing the report to show (albeit a blank box) even when there is not one (i.e. when job is NEW or IN_PROGRESS)?